### DISCUSSION TITLE
Is it possible to retrieve "Shadow/System" Tiddler using "WebServer API"?

### DISCUSSION THREAD

**Discussion by msameen99** [TiddlyWiki/TiddlyWiki5#6866] [Q&A] [answered]
Hi,

I’m using the [WebServer API: Get All Tiddlers](https://tiddlywiki.com/#WebServer%20API%3A%20Get%20All%20Tiddlers) successfully to retrieve my Non-System Tiddlers with some customized filters (after creating the “$:/config/Server/AllowAllExternalFilters” Tiddler), But it does NOT retrieve the “System Tiddlers”.

So, Is there ANY way to retrieve “System Tiddlers”?

My use case:
I’ve very big number of Special Tiddlers (contains Meta Data of my TiddlyWiki file) which I make their titles start with “$:/” so it can be “hidden” or different from my normal Tiddlers, Now I need to get those tiddlers using Web-Server API.

Note: The above link shows that the Response Body of the above API is an array of all non-system tiddlers, is that mean there is no way to retrieve “System Tiddlers”?

Regards

**Comment by pmario** [comment]
You can try a filter like this:  `[all[shadows+tiddlers]]` see: https://tiddlywiki.com/prerelease/#list-links-draggable%20Macro

**Reply by msameen99** [reply]
I already tried `[all[shadows+tiddlers]]` , but the API still not retrieve my tiddlers that start with **"$:/"**.
this actually match what in the documentation :-
![image](https://user-images.githubusercontent.com/16881960/182100167-6699440f-26f4-42d5-ae03-70ddf3525f0f.png)

and I'm wondering if there is a way to overcome this limitation

**Comment by saqimtiaz** [comment, marked as answer]
Try setting `$:/config/SyncSystemTiddlersFromServer` to `yes`. Note that this will mean that tiddlers like `$:/StoryList` will also sync between any connected clients.

**Reply by msameen99** [reply]
Thanks a MILLIONS Saqimtiaz, it WORKS

regarding the sync of "`$:/StoryList`" (I didn't know much about it) is that a problem? what should I keep eyes on? 

Thanks again

**Reply by saqimtiaz** [reply]
It is problematic if you have multiple users connecting to the same node.js TW server, as changes to one users story list get propagated to other users.

**Comment by Jermolene** [comment]
I updated the docs to add a note about syncing system tiddlers:

fe74a776e9a8dfc8821d5028777028f76fe42783
